test/jdk/java/awt/FontClass/DrawStringWithInfiniteXform.java

```diff
@@ -58,11 +65,15 @@ public static void main(String [] args) {
     }
 
     private void start() {
+        System.out.println("start at " + System.currentTimeMillis());
+        System.out.flush();
         float[] vals = new float[6];
-        for (int i=0;i<6;i++) vals[i]=Float.POSITIVE_INFINITY;
+        for (int i=0; i<6; i++) {
```

Comment:
We can add proper indentation for operators before push, since we are updated at other places.